src/ioctl.rs

```diff
@@ -55,6 +90,15 @@ macro_rules! ioctl_io_nr {
 }
 
 /// Declare an ioctl that reads data.
```

Comment:
Let's keep the same way of talking about what a macro does across the whole module. The other macro definitions (in vmm-sys-utils) are written as:
```A macro for declaring an [`ioctl`](http://man7.org/linux/man-pages/man2/ioctl.2.html) that reads data.```

For making it extra neat I would phrase them all the same.